mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,311 @@
+# Bluetooth Mapper
+
+
+## Introduction
+
+Mapper is an application that is used to connect  and control devices. This is an implementation of mapper for 
+bluetooth protocol. The aim is to create an application through which users can easily operate  devices using bluetooth protocol
```

Comment:
nit: please remove the extra whitespace between `operate` & `devices`

Reply:
done, thanks